### DISCUSSION TITLE
Remove default Tailwind classes?

### DISCUSSION THREAD

**Discussion by boutzamat** [gilbitron/laravel-vue-pagination#159] [Feature requests]
The pagination has some default CSS classes out of the box when using Tailwind.
This means you have to use !important classes to override these classes.
Would be nice if:

- No default classes was added. This is Tailwind after all, so if you want to design a pagination, some level of Tailwind is expected.
- Or if a custom class is added, it doesn't append but instead replaces the default classes

**Comment by gilbitron** [comment]
You can override the Tailwind classes by using the `item-classes` and `active-classes` props. Each takes an array of class names. See https://laravel-vue-pagination.org/guide/components/tailwind.html
